src/conduits/RwaInputConduit3.sol

```diff
@@ -197,28 +197,64 @@ contract RwaInputConduit3 {
     //////////////////////////////////*/
 
     /**
-     * @notice Method to swap USDC contract balance to DAI through PSM and push it into RwaUrn address.
-     * @dev `msg.sender` must first receive push acess through mate().
+     * @notice Swaps the GEM balance of this contract into DAI through the PSM and push it into the `to` address.
+     * @dev `msg.sender` must have received push access through `mate()`.
      */
     function push() external isMate {
-        uint256 balance = gem.balanceOf(address(this));
+        _doPush(gem.balanceOf(address(this)), 0);
+    }
+
+    /**
+     * @notice Swaps the specified amount of GEM into DAI through the PSM and push it into the `to` address.
+     * @dev `msg.sender` must first receive push acess through mate().
```

Comment:
Use the same message as the method above for consistency.
```suggestion
     * @dev `msg.sender` must have received push access through `mate()`.
```

Reply:
Done